main.js

```diff
@@ -1934,8 +1934,8 @@ async function checkWeatherVariables() {
         });
         await adapter.setObjectNotExistsAsync('forecast.current.observationTime', {
             type: 'state',
-            common: {name: 'Observation time (rfc822)', role: 'date', type: 'string', read: true, write: false},
-            native: {id: 'current_observation.local_epoch'}
+            common: {name: 'Observation time (rfc822)', role: 'value.datetime', type: 'string', read: true, write: false},
```

Comment:
value.datetime is a deprecated role ... so "date" is the correct one, see https://github.com/ioBroker/ioBroker/blob/master/doc/STATE_ROLES.md#common